lib/ast_nodes.py

```diff
@@ -31,19 +46,62 @@ def evaluate(self, steps_range, context=dict()):
                 result = f'[{result}::{new_steps_range[1] - 1}]'
 
             return result
+
         else:
-            for expression in self.expressions:
-                result += f'{expression.evaluate(steps_range, context)}:'
+            assert False
+
+    def get_interpolation_conditioning(self, model, get_learned_conditioning, steps_range, context=None):
+        from lib.interpolation_conditioning import InterpolationConditioning
+
+        total_steps = steps_range[1]
+
+        conditionings = []
+        for expression in self.expressions:
+            prompt = expression.evaluate(steps_range, context)
+            conditionings.append(get_learned_conditioning(model, [prompt], total_steps)[0][0].cond)
```

Comment:
get first cond function pour intention

Reply:
we don't know yet how to implement this. maybe we need to `get_learned_conditioning` on every prompt before making them all the same dimensions. resolving this as it would need more than just replacing this line with a function call